widgetssdk/src/main/java/com/glia/widgets/core/queue/QueueRepository.kt

```diff
@@ -94,11 +128,18 @@ internal class QueueRepository(private val gliaCore: GliaCore, private val confi
 
         currentQueues[index] = queue
 
-        _observableIntegratorQueues.onNext(QueuesState.Queues(currentQueues.toList()))
+        _queuesState.onNext(QueuesState.Queues(currentQueues.toList()))
     }
 
-    private fun mapCurrentQueues(siteQueues: List<Queue>, queueIds: Data<List<String>>): List<Queue> = when (queueIds) {
-        is Data.Value -> siteQueues.filter { queueIds.result.contains(it.id) }
-        else -> siteQueues.filter { it.isDefault == true }
+    private fun matchQueues(queueIds: List<String>, siteQueues: List<Queue>) {
+        val matchedQueues = siteQueues.filter { queueIds.contains(it.id) }
+
+        if (matchedQueues.isEmpty()) {
+            setDefaultQueues(siteQueues)
+        } else {
+            Logger.i(TAG, "Setting up queues. Matched ${matchedQueues.count()} Queues out of ${queueIds.count()}")
```

Comment:
What do you think regarding this wording?
`Setting up queues. ${matchedQueues.count()} out of ${queueIds.count()} queues provided by an integrator match with site Queues.`
I'm suggesting it because it's more self-explanatory even for people who will see it for the first time, hopefully. 🙂